simulation-runner/config/nodes/IrvingFactsChange.json

```diff
@@ -0,0 +1,7 @@
+{
+  "simulation_class": "com.puppetlabs.gatling.node_simulations.IrvingFactsChange",
+  "certname_prefix": "gat-agent-",
```

Comment:
If we use "perf-agent-" instead of "gat-agent-" here, it would be simpler to use with our existing scripts, we could just override the config/node/*.json path and use as is.